server/src/server/init-server.js

```diff
@@ -14,6 +14,7 @@ const staticFrontendRoutes = require('../plugins/staticFrontendRoutes')
 const apiRoutes = require('../plugins/apiRoutes')
 const errorMapper = require('../plugins/errorMapper')
 const queryFilter = require('../plugins/queryFilter')
+const pagination = require('../plugins/paginationPlugin')
```

Comment:
Yes, that would make it more consistent. I'm a bit on the fence with these naming conventions though, as it is easier to quickly navigate to if it's named with eg. "plugin" in the name. I've noticed that there are a lot of files names eg. "organisation", which are in different folders and it makes it a bit harder to navigate to compared to if they were named "organisationService" and "organisationModel".

Reply:
Staying consistent within the current architecture is important, and if we want to change it we should do so across the board in a PR so that everything is in a new consistent state.

There is little value in having two different naming schemes within the same code base, as on a cursory glance, how would a new developer know which of the naming schemes is the proper one?

They would have to browse the history to figure that out, and even then, they may arrive at a different preference.

Reply:
I agree, it was just something I tried out - will change it to be more consistent.

Reply:
Great, thanks.